src/extension.ts

```diff
@@ -42,24 +44,127 @@ export function activate(context: vscode.ExtensionContext) {
 
     const fileStrArr = fileStr.split('\n').filter((line) => line.trim());
     let arrResult = readArrayLines(fileStrArr);
+    let typesMap = new Map<string, Array<number>>();
     for (let i = 0; i < arrResult.length; i++) {
       if (reservedMainKeywords.includes(arrResult[i].line[0].value)) {
         arrResult[i].mainType = arrResult[i].line[0].value;
       } else {
         arrResult[i].mainType = "attribution";
       }
+
+      let currValue: Array<any> = [];
+
+      if(typesMap.has(arrResult[i].mainType)) {
+        console.log("Map content to be retrieved: ", typesMap.get(arrResult[i].mainType));
```

Comment:
I believe these lines may have been used for debug purposes, but are not needed.
For readability purposes, we could change this into comments.